dsrc_driver/CMakeLists.txt

```diff
@@ -123,6 +123,7 @@ catkin_package(
 include_directories(
  include
  thirdparty/rapidjson/include
+ ${cav_driver_utils_INCLUDE_DIRS}
```

Comment:
I think this should be included as part of catkin_INCLUDE_DIRS 
Does it not compile without it?